app/assets/javascripts/admin/modules/ga4-page-view-tracking.js

```diff
@@ -0,0 +1,15 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {}
+;(function (Modules) {
+  function GA4PageViewTracking(module) {
+    this.module = module
+  }
+
+  GA4PageViewTracking.prototype.init = function () {
+    this.trackingData = this.module.getAttribute('data-attributes')
+    window.dataLayer = window.dataLayer || []
+    dataLayer.push(JSON.parse(this.trackingData))
```

Comment:
did this not work? `window.GOVUK.analyticsGa4.core.sendData(data)`
do we need to push it directly to datalayer for it to appear as one?
Figuring if we use the publishing component one then if they make any changes or fixes we get it for free

Reply:
If we use this, then we can get rid of the dataLayer exception in eslint as well